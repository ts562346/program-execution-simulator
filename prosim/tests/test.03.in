test3 3
BLOCK 3
BLOCK 5
HALT

