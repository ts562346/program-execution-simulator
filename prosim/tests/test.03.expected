00000: BLOCK
00003: BLOCK
00008: HALT

DOOP count : 0
DOOP time  : 0
BLOCK count: 2
BLOCK time : 8
